### ISSUE TITLE
Resolving df-py for existing data and missing data when running all.sh

### ISSUE THREAD

**Issue by idiom-bytes** [oceanprotocol/df-py#275]
**[Expected Data is Missing - First Run]**
We are going to resolve df-py for 3 cases:
1. The case where there is missing FREs or basic data, it will fail, report an error, and we will have to seed the data manually.
1. The case where there are FREs, consume, allocate, and everything exists, we expect the csvs to be generated with the right amounts.
1. In the case where there is FREs, but missing consume, or allocate, or some other data, we should generate empty data or zeroes, so that df-py completes, dfsql is updated, and df-web can run normally.

#1 - We agreed that is just going to fail.
#2 - Is the current happy-path that we're expecting at the moment.
#3 - Is the concern, that in the beginning of each epoch the dashboard is not going to work (consume/estimated rewards will be at 0)

In cases #2 and #3, we should have tests that cover both paths, and verify that they are working, or at least have tested and verified that this is working as expect.

**[Raising an Error]**
If there is no data, we should raise an error. 
If we raise an error, then the remainder of the all.sh pipeline fails, and it will use the previously generated CSV rather than null. In this case, we can see the error logs.

End Scenario: dfsql would get from latest update, before the error, therefore the data would look the same. The data would not be updated.

**[Breakdown by CSV]**
- getRates.csv will fail pipeline if error. Should be populated automatically.
- nftVols.csv will complete if there is missing consume, but will fail if there is missing assets/nfts. If there are missing assets, **it should be populated manually.**
- approved.csv should always complete
- symbols.csv should always complete, and it's dependant on the approved tokens.
- veOcean.csv is going to fail if we don't have any locks, **populate manually**.
- allocations.csv is going to fail if we don't have any allocations, **populate manually**.
- rewardsPerLP.csv should always pass if the previous files and dependencies have been addressed. However, it's possible that it will fail if someone has allocated, but they have no veOCEAN. In this case, we would **populate this manually**.

TLDR #1; dataNFTs, veOcean locks, and allocations are the only things we will need to seed manually to have the pipeline pass.
TLDR #2: Core systems expect Allocator + Locker to be valid. We need someone with veOCEAN who has allocated in order to properly calculate rewardsPerLP.csv. It could be that this fails if this statement is not true.
